### ISSUE TITLE
No longer working since last update

### ISSUE THREAD

**Issue by atr00** [cellcortex/homebridge-yeelighter#54]
Hi,

I am facing issue since the last update. It is no longer working.
I can slide the light power but it won't even turn on.
The custom name that I set in Home will also always gets reset.

I can see it in the console:
```
[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 2 main#main] setConfiguredName [ 'GuangCan Ceiling Light 2 main' ]
[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 2 main#main] setConfiguredName [ 'GuangCan Ceiling Light 2 main' ]
[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 2 main#main] setConfiguredName [ 'GuangCan Ceiling Light 2 main' ]

[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 3#main] setConfiguredName [ 'GuangCan Ceiling Light 3 main' ]

[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 3 main#main] setConfiguredName [ 'GuangCan Ceiling Light 3 main' ]

[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 3 main#main] setConfiguredName [ 'GuangCan Ceiling Light 3 main' ]
[12/2/2020, 12:56:36 PM] [Yeelighter] [GuangCan Ceiling Light 3 main#main] setConfiguredName [ 'GuangCan Ceiling Light 3 main' ]
[12/2/2020, 12:56:40 PM] [Yeelighter] [0x00000000101cc163] ceiling20 Connected []
[12/2/2020, 12:56:40 PM] [Yeelighter] [0x00000000101cc163] requesting attributes. Transactions: 5 []
[12/2/2020, 12:56:40 PM] [Yeelighter] [GuangCan Ceiling Light 3 main#main] temperature light updated {"power":true,"color_mode":2,"bright":94,"hue":0,"sat":0,"ct":2703,"bg_power":false,"bg_bright":100,"bg_hue":6,"bg_sat":100,"bg_ct":4000,"bg_lmode":3,"nl_br":22,"active_mode":1,"name":""} []
```

I tried to removed all cached accessories, restart, etc.

**Comment by Sungamm** [comment]
Had the same isse. Try to go back to 1.5.16 - for me its works. If you have some strange device left (from 2.1.11) use 

sudo rm -rf /var/homebridge/accessories

Good luck.

**Comment by cellcortex** [comment]
After going back and forth, I believe this is finally fixed in 2.3.4, Unfortunately you may have to delete the accessory cache once more.
